packages/odnnr/odnnr.2.0.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "https://github.com/UnixJunkie/odnnr"
+bug-reports: "https://github.com/UnixJunkie/odnnr/issues"
+dev-repo: "git+https://github.com/UnixJunkie/odnnr.git"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+install: ["R" "CMD" "BATCH" "install_keras.r"]
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml"
+  "conf-r"
+  "dune" {>= "1.0.0"}
```

Comment:
The dune-project file requires dune 1.11. I can modify the file unless you want to lower the bound in the sources.
```suggestion
  "dune" {>= "1.11.0"}
```